src/__screenshot_tests__/data-card-screenshot-test.tsx

```diff
@@ -93,3 +94,16 @@ test('DataCard without icon, with top actions and too long title', async () => {
 
     expect(image).toMatchImageSnapshot();
 });
+
+test.each(ASPECT_RATIO_VALUES)('DataCard with aspect ratio %s', async (aspectRatio) => {
+    await openStoryPage({
+        id: 'components-cards-data-card--default',
+        args: {aspectRatio: aspectRatio.replace(':', ' ')},
+    });
```

Comment:
I'd use mobile viewport in this test. The resulting screenshots would be les weird

Reply:
done